src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -0,0 +1,203 @@
+
+    <img
+        title="Vulnerability instance details form"
+        alt="An image showing the ignore form on vulnerability instance details page"
+        src={vmIgnoreEntityForm}
+    />
+
+    </Collapser>
+    <Collapser
+        id="bulk-ignore"
+        title="Ignore a vulnerability across multiple instances"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 
+1. Click the vulnerability you want to ignore. By default, this vulnerability view is filtered to only show vulnerabilities where at least one entity is marked Affected.
+
+    <img
+        title="Vulnerabilities tab for all entities"
+        alt="An image showing the all entities vulnerabilities tab of Vulnerability Management"
+        src={vmIgnoreBulkVulnList}
+    />
+
+1. Referring to **Area of Impact**, select each entity you want to ignore, then click the **Change to ignored** button.
+    
+    <img
+        title="Vulnerabilities details"
+        alt="An image showing the selection of specific entities"
+        src={vmIgnoreBulkDetails}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Ignored. Include a time period that the vulnerability should be ignored.
+
+        <img
+            title="Vulnerabilities details form "
+            alt="An image showing the ignore form on vulnerability instance details page"
+            src={vmIgnoreBulkForm}
+        />
+    </Collapser>
+</CollapserGroup>
+
+## Change Ignored status to Affected [#bulk-manage]
+
+It's good practice to review ignored vulnerabilities and validate whether they should stay ignored. Changing the status from Ignored to Affected reintroduces the vulnerability back into default views andcounts on this entity, but other instances of this vulnerability on other entities will still remain ignored.
+
+<CollapserGroup>
+    <Collapser
+        id="remove-ignored"
+        title="Remove ignored status for a specific entity"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select the entity you want to update) > **Vulnerability Management** > Click the **Vulnerabilities** tab.
+1. View all your ignored vulnerabilities by updating the filter from `Status = Affected` to `Status = Ignored`. 
+
+    <img
+    title="Vulnerabilities tab scoped to a specific entity"
+    alt="An image showing the scoped vulnerabilities tab of Vulnerability Management with filter tab open"
+    src={vmAffectedEntityVulnList}
+    />
+
+1. Choose the vulnerability that you want to update, then click **Change to affected**.
+
+    <img
+        title="Vulnerabilities tab scoped to a specific entity and only ignored instances"
+        alt="An image showing the scoped vulnerabilities tab of Vulnerability Management filtered to only ignored instances"
+        src={vmAffectedEntityFiltered}
```

Comment:
Maybe this should be the other visual vmAffectedEntityDetails
The image you are using here is still on the vulnerability list with it filtered down but not the one with the 'Change to affected' button?

Reply:
Updated!